modelmapper/mapper.py

```diff
@@ -158,11 +161,22 @@ def hastype(self):
     SqlalchemyFieldType.Integer,
     SqlalchemyFieldType.BigInteger
 }
-NUMERIC_REMOVE = (',', '$', '%')
+
+NUMERIC_REMOVE = (',', '$', '%', '-')
+
+
+def _remove_extra_chars_from_number(item, absolute=False):
```

Comment:
I think the absolute kwarg would be more understandable as `abs` (standard library func name) or `absolute_value` (verbose).